### PR TITLE
Eslint fixes

### PR DESCRIPTION
**Pull request by Tomcatus**

### CODE REVIEW

File: app/components/AddressBlock.js [hunk 1]
```diff
@@ -70,48 +74,48 @@ const NoSmartAddressesContainer = styled.div`
...
 
 type Props = {
   history: object,
   accountBlock: Object, // TODO: type this
   syncAccountOrIdentity: Function,
   selectedAccountHash: string,
   accountIndex: number,
-  theme: Object
+  theme: Object,
+  history: Object
```
Comment (umurb) [inline app/components/AddressBlock.js]: Duplicate history

File: app/components/AddressBlock.js [hunk 2]
```diff
@@ -238,69 +232,68 @@ class AddressBlock extends Component<Props, State> {
...
-              Add a Delegate
+            <NoSmartAddressesTitle>Delegation Tips</NoSmartAddressesTitle>
+            {this.renderNoSmartAddressesDescription(
+                  noSmartAddressesDescriptionContent
+                )}
+            <NoSmartAddressesButton
+              small
+              buttonTheme="secondary"
+              onClick={this.openDelegateModal}
+            >
```
Comment (umurb) [inline app/components/AddressBlock.js]: missing disabled={!isManagerReady}

File: app/components/EmptyState/index.js [hunk 3]
```diff
@@ -11,37 +11,34 @@ const Container = styled.div`
...
-`
+`;
 
 type Props = {
   title: string,
-  desription: React.Node
-}
+  description: React.Node,
+  imageSrc: string,
+  className: string
```
Comment (umurb) [inline app/components/EmptyState/index.js]: className: string - crashes the app, resulting in empty screen.

classname-string

File: app/components/AddressBlock.js (hunk 2 above)
Comment (Tomcatus) [reply app/components/AddressBlock.js]: Fixed.

File: app/components/AddressBlock.js (hunk 1 above)
Comment (Tomcatus) [reply app/components/AddressBlock.js]: Fixed.

File: app/components/EmptyState/index.js (hunk 3 above)
Comment (Tomcatus) [reply app/components/EmptyState/index.js]: Fixed.

**Pull request merged**